infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/LogReplicationConfig.java

```diff
@@ -111,6 +111,8 @@ public LogReplicationConfig(Map<ReplicationSubscriber, Set<String>> subscriberTo
 
     // An enum of all supported Replication Models
     public static enum ReplicationModel {
-        SINGLE_SOURCE_SINK
+        SINGLE_SOURCE_SINK,
+        DYNAMIC_DESTINATION,
+        ROUTING_QUEUE
```

Comment:
We can have the replication  model as a proto msg and use that proto msg everywhere.
We can keep it this way in this PR and I can take care of it in mine as I already have the replicationModel as a proto msg even in the replicationSession.

I also think we can have the replicationSession as a proto msg itself. We can take it as a separate task when time permits.

No action needed, just wanted to share the thought

Reply:
Yes I agree.  Since the change to use protobufs for the models and session is in progress in your and Anny's PRs, I did not add it here.

Reply:
+1 

I also have these two in my PR as a proto messages.. what if we merge at least the replication models as protoBuf so all these PRs reuse that? (no need in this PR to change the ReplicationSession for the proto as that is a major change and I'm taking care of that in my PR)